virt/virt-4-11-release-notes.adoc

```diff
@@ -130,6 +128,10 @@ To preserve and support your existing nmstate configuration, install the xref:..
 
 //CNV-18323 VM templates can be made public i.e. available to all users
 
+* As of version 4.11, `node-maintenance-operator` (NMO) is no longer shipped with {VirtProductName}. It is now available to deploy as a standalone operator through the Operator Lifecycle Manager (OLM) in the OperatorHub. 
++
+If you are using OpenShift Virtualization 4.10 or earlier, you must move all nodes out of maintenance mode prior to upgrading to OpenShift 4.11. If a node must stay in maintenance mode during the upgrade, the cluster admin must install the standalone NMO and replace the `nodemaintenances.nodemaintenance.kubevirt.io` CR with a nodemaintenances.nodemaintenance.medik8s.io CR before upgrading to OpenShift Virtualization 4.11. You will receive an error and be unable to upgrade if you attempt to upgrade with a `nodemaintenances.nodemaintenance.kubevirt.io` CR.
```

Comment:
Can you please reword this? `You will receive an error and be unable to upgrade if you attempt to upgrade with a `nodemaintenances.nodemaintenance.kubevirt.io` CR.`.

Reply:
@dbasunag Can you clarify what's incorrect here so I can reword it?

Reply:
`You will receive an error` - an alert/event would be generated advising the user that upgrade is blocked when NMO is in use.  So I feel, we should remove this and indicate that upgrade to 4.11 would be blocked if non-standalone NMO CR is in use.